source/routing/defining-your-routes.md

```diff
@@ -198,6 +198,96 @@ Visiting `/posts` is slightly different. It will first render the
 Finally, visiting `/posts/new` will first render the `posts` template,
 then render the `posts/new` template into its outlet.
 
+#### Resetting Nested Route Namespace
+
+When nesting routes, it may be beneficial for a child route to not inherit its ancestors name. This allows you to reference and reuse a given route in multiple route trees as well as keep the class name short. This is equivalent to the behavior of the now deprecated `this.resource()` function.
```

Comment:
> as keep the class name short

Minor nitpick: in modules world the issue is with directory nesting not class name (as the actual class name does not ever need to be typed out).